source/json/highlights.json

```diff
@@ -1,11 +1,11 @@
 [
   {
-    "title": "Mozilla Global Sprint",
-    "image": "https://assets.mofoprod.net/network-pulse/images/entries/2017-04-17_195731.4603530000.png",
-    "description": "Mozilla’s Global Sprint is a fun, fast-paced and two-day collaborative event to hack and build projects for a healthy Internet. A diverse network of scientists, educators, artists, engineers and others come together in person and online to innovate in the open.",
-    "link_url": "https://mozilla.github.io/global-sprint/",
-    "link_label": "Join This Project",
-    "footer": "<a class=\"btn btn-ghost\" href=\"/projects\">See More Projects</a>"
+    "title": "MozFest 2017 Call for Proposals",
+    "image": "https://network.mofoprod.net/_images/upcoming/mozfest-call-for-proposal.jpg",
```

Comment:
If you want to use an image that's committed to this repo, then you can leave out the domain and just put:

`"image": "/_images/upcoming/mozfest-call-for-proposal.jpg",`